test.js

```diff
@@ -22,3 +23,29 @@ test('catch', async t => {
 
 	t.true(inRange(promise.time, 200, 250));
 });
+
+test.cb.serial('log', t => {
+	function myDelay(ms) {
+		return new Promise(resolve => {
+			setTimeout(resolve, ms);
+		});
+	}
+
+	const unhook = hookStd.stdout(output => {
+		unhook();
```

Comment:
Unrelated, but it hit me that it might be nice for `hookStd` to have a `once` option so we didn't have to manually `unhook()`. Also wonder why I didn't make it a Promise interface. What do you think?

This could be:

```js
test.serial('log', async t => {
	function myDelay(ms) {
		return new Promise(resolve => {
			setTimeout(resolve, ms);
		});
	}

	const promise = hookStd.stdout();
	m.log(myDelay)(200);
	t.regex(await promise, /Promise from myDelay resolved in [0-9]+ ms/);
});
```

So much nicer. No callback test.